js/components/EventsDataFetcher.js

```diff
@@ -0,0 +1,43 @@
+import React, { useState, useEffect } from 'react';
+import Events from './Events';
+
+const EventsDataFetcher = () => {
+  // Based off https://reactjs.org/docs/faq-ajax.html
+  
+  const [error, setError] = useState(null)
+  const [isLoaded, setIsLoaded] = useState(false)
+  const [events, setEvents] = useState([])
+  
+  // Note: the empty deps array [] means
+  // this useEffect will run once
+  // similar to componentDidMount()
+  useEffect(() => {
+    fetch('https://newsroom.eclipse.org/api/events?parameters[upcoming_only]=1')
```

Comment:
We should make the retrieval of upcoming only a parameter with a default value rather than hard code it. There are cases where we might want to display past events for users. This may include cases like to see events they may have been at, or to see how an event they are interested in went a previous year.

Reply:
Yep, will be my TODO.